### ISSUE TITLE
update credits dialog

### ISSUE THREAD

**Issue by incanus** [mapbox/DEPRECATED-mapbox-ios-sdk#542]
This needs a visual refresh. Image, date, etc. 

![](https://pbs.twimg.com/media/B3E49RFIAAATrAI.png)

**Comment by ljbade** [comment]
Isn't that the old logo?

Perhaps a white on blue theme would be more interesting.

**Comment by incanus** [comment]
Yeah, we didn't really get a new logo, so I left it some time back. This view controller takes on the map view's `tintColor` property and colorizes the text and logo on the fly to match. It's the default controller for the bottom-right _i_ button so that people accredit OSM & our software property even if they do nothing else. 

I will probably just change this to the word mark and tweak things a bit.
